src/Mpdf.php

```diff
@@ -22271,7 +22271,10 @@ function _tableGetHeight(&$table, $i, $j)
 			$h = $heightrow[$i];
 			if (isset($cell['rowspan'])) {
 				for ($k = $i + $cell['rowspan'] - 1; $k > $i; $k--) {
-					$h += $heightrow[$k];
+					if (array_key_exists($k, $heightrow)) {
+                        $this->logger->debug('Possible non-wellformed HTML markup in a table', ['context' => LogContext::HTML_MARKUP]);
+                        $h += $heightrow[$k];
```

Comment:
Also, indents. And, the log message should be in the else branch as I mentioned previously. Like this, it makes no sense.